src/views/RecIM/components/List/Listing/index.js

```diff
@@ -136,22 +137,47 @@ const ActivityListing = ({ activity }) => {
   );
 };
 
-const TeamListing = ({ team }) => {
+const TeamListing = ({ team, invite }) => {
   if (!team) return null;
+
+  const handleAcceptInvite = async () => {
+    await respondToTeamInvite(team.ID, { Response: "accepted"})
+  }
+
+  const handleRejectInvite = async () => {
+    await respondToTeamInvite(team.ID, { Response: "rejected" })
+  }
+
   return (
     <ListItem key={team.ID}>
       <ListItemButton
         component={Link}
         to={`/recim/activity/${team.Activity.ID}/team/${team.ID}`}
         className={styles.listing}
       >
-        <Grid container columnSpacing={2}>
-          <Grid item xs={12} sm={8}>
-            <Typography className={styles.listingTitle}>{team.Name}</Typography>
-          </Grid>
-          <Grid item xs={12} sm={4} className={styles.rightAlignLarge}>
-            <Typography className={styles.listingSubtitle}>{team.Activity?.Name}</Typography>
+        <Grid container rowSpacing={1}>
+          <Grid container columnSpacing={2}>
+            <Grid item xs={12} sm={8}>
+              <Typography className={styles.listingTitle}>{team.Name}</Typography>
+            </Grid>
+            <Grid item xs={12} sm={4} className={styles.rightAlignLarge}>
+              <Typography className={styles.listingSubtitle}>{team.Activity?.Name}</Typography>
+            </Grid>
           </Grid>
+          {invite ? (
+            <Grid container columnSpacing={2}>
+              <Grid item>
+                <IconButton className={styles.greenButton} onClick={handleAcceptInvite}>
+                  <CheckIcon />
+                </IconButton>
+              </Grid>
+              <Grid item>
+                <IconButton className={styles.redButton} onClick={handleRejectInvite}>
+                  <ClearIcon />
+                </IconButton>
+              </Grid>
+            </Grid>
+          ) : null}
```

Comment:
```suggestion
          {invite && (
            <Grid container columnSpacing={2}>
              <Grid item>
                <IconButton className={styles.greenButton} onClick={handleAcceptInvite}>
                  <CheckIcon />
                </IconButton>
              </Grid>
              <Grid item>
                <IconButton className={styles.redButton} onClick={handleRejectInvite}>
                  <ClearIcon />
                </IconButton>
              </Grid>
            </Grid>
          )}
```

Reply:
@cpabbot found a relatively nifty way of returning jsx without having to ternary into null

Reply:
Personally, I think that this can be a little confusing and overall is not done much in 360.

Reply:
Yes, it's a nice little trick. We use this here and there throughout the repo